plugin.video.svtplay/addon.xml

```diff
@@ -0,0 +1,29 @@
+<addon id="plugin.video.svtplay" name="SVT Play" provider-name="nilzen" version="5.1.12+matrix.1">
+  <requires>
+    <import addon="script.module.requests" version="2.22.0" />
```

Comment:
@enen92 Will this be a problem in Matrix? Do I need to update this myself somehow or should `kodi-addon-submitter` take care of this? 🤔

Reply:
`2.22.0+matrix.1` seems to be the version I need according to https://github.com/xbmc/repo-scripts/blob/matrix/script.module.requests/addon.xml ❓

Reply:
It's fine as is. The version tag for a dependency specifies the minimum version required (2.20.0+matrix.1 is > than 2.20.0). When Kodi migrates the add-ons it starts by migrating the dependencies first, so you'll receive the matrix version of requests before receiving the version of stvplay. This was fixed and improved in https://github.com/xbmc/xbmc/pull/17703 if you're curious about the implementation.
Cheers